app/models/events/event_record.rb

```diff
@@ -0,0 +1,9 @@
+module Events
+  ##
+  # This Module adds common method to all ApplicationRecord modeling Events
+  module EventRecord
+    def event_name
+      self.class.name.demodulize.underscore
+    end
+  end
+end
```

Comment:
Wouldn't be better to create a helper to do this?

Reply:
Done it. Thanks!